docs/getstarted/userinterface.md

```diff
@@ -279,6 +279,38 @@ You can Drag and Drop editor groups on the workbench, move individual Tabs betwe
 
 >**Note:** VS Code uses editor groups whether or not you have enabled Tabs.  Without Tabs, editor groups are a stack of your open items with the most recently selected item visible in the editor pane.
 
+### Locked editor groups
+
+By default, files will always open in the active editor group. Locked editor groups help to avoid this. The `workbench.editor.autoLockGroups` setting allows you to select editor types that should lock a group automatically when they open. This only applies when the editor is the first to open in an otherwise empty or new group.
+
+```json
+"workbench.editor.autoLockGroups": {
+  "default": false,
+  "terminalEditor": true
+}
+```
+
+Terminals are configured by default to cause a new group to lock automatically. A lock icon in the action toolbar (top right) indicates a locked group. The locked state of an editor group is persisted and restored across restarts.
+
+<!-- Required? Locked editor https://code.visualstudio.com/assets/updates/1_61/editor-readonly-deleted.png -->
```

Comment:
still open. If needed, can someone move the images to the correct location and create the image link?